examples/daemonsetsharding/deployment-no-node-pods.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: apps/v1
```

Comment:
This file should be generated from jsonnet See: https://github.com/kubernetes/kube-state-metrics/blob/main/jsonnet/kube-state-metrics/kube-state-metrics.libsonnet

Reply:
> This file should be generated from jsonnet See: https://github.com/kubernetes/kube-state-metrics/blob/main/jsonnet/kube-state-metrics/kube-state-metrics.libsonnet

fix